mhs/common/mhs_common/workflow/synchronous.py

```diff
@@ -69,42 +68,39 @@ def __init__(self,
                                                                                 interaction_details=interaction_details,
                                                                                 message=payload)
         except Exception:
-            logger.error('002', 'Failed to prepare outbound message')
+            logger.error('Failed to prepare outbound message')
```

Comment:
`exception()` ?

Reply:
The original logging didn't log exception so I've left it the same